src/features/common/Layout/CookiePolicy/index.tsx

```diff
@@ -7,6 +7,13 @@ export default function CookiePolicy() {
   const [showCookieNotice, setShowCookieNotice] = useState(true);
   const { useTranslation } = i18next;
   const { t } = useTranslation(['leaderboard']);
+
+  const [initialized, setInitialized] = React.useState(false);
+
+  React.useEffect(() => {
+    i18next.initPromise.then(() => setInitialized(true));
+  }, []);
```

Comment:
I assume we cannot reuse the initialized state created here? 
https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/c21ddeee8d189a26f8ed58261759a1e15deff4a3/pages/_app.tsx#L45-L49
Are several initializations of i18n no problem?

Reply:
- I tested i18n in debug mode and it does not seem to be a problem call i18next.initPromise several times. 
- I also think it not worth the effort to reach down the initialized values by properties through the Layout into the CookiePolicy components and similar for the NavBar - maybe once we could think of using i18n as HOC?